models/git/commit_status.go

```diff
@@ -191,6 +192,11 @@ func (status *CommitStatus) APIURL(ctx context.Context) string {
 	return status.Repo.APIURL() + "/statuses/" + url.PathEscape(status.SHA)
 }
 
+// LocaleString returns the locale string name of the Status
+func (status *CommitStatus) LocaleString(lang translation.Locale) string {
+	return lang.Tr(status.State.String())
```

Comment:
ps: without a proper prefix and comments, it's not friendly to the locale system, eg: it's unsearchable and it's unclear what keys it needs.

@silverwind

Reply:
We can't import translation in `modules/structs/commit_status.go`. (import loop)

Not sure whether this is good: move `structs.CommitStatusState` to `git.CommitStatusState`